### PR TITLE
Update actions in TM to use stringify Refs #6467

### LINKED ISSUE #6467: Update stringstream to stringify in Tensor Mechanics Actions
### Description of the enhancement
A majority of the actions within Tensor Mechanics use std::stringstream to create unique kernel names; these actions ought to use Moose::stringify.

### Rationale for the enhancement
As identified by @dschwen in a previous pull request, these actions should use stringify to be up-to-date and avoid acting as poor examples for future code development

### Identified impact
Update string creation for unique kernel names in PoroMechanicsAction, PressureAction, and TensorMechanics Action.  No API changes.

### PR DESCRIPTION
**Pull request by sapitts**
- Changed the tensor mechanics actions to use Moose::stringify instead of std::stringstream
- Corrected spelling mistakes in the RSpherical and AxisymmetricRZ actions
- Used the atom option 'Show Invisibles' so that I could count dots for the spaces
- Existing test cases
- Closes #6467

### CODE REVIEW

File: modules/tensor_mechanics/include/actions/PoroMechanicsAction.h [hunk 1]
```diff
@@ -22,7 +22,6 @@ class PoroMechanicsAction : public TensorMechanicsAction
   virtual void act();
 
 private:
```
Comment (dschwen) [inline modules/tensor_mechanics/include/actions/PoroMechanicsAction.h]: Might as well remove these two lines, too.

File: modules/tensor_mechanics/src/actions/TensorMechanicsAction.C [hunk 2]
```diff
@@ -31,7 +32,8 @@ InputParameters validParams<TensorMechanicsAction>()
 }
 
 TensorMechanicsAction::TensorMechanicsAction(const InputParameters & params) :
-    Action(params)
+    Action(params),
+    _base_name(isParamValid("base_name") ? getParam<std::string>("base_name") + "_" : "")
```
Comment (dschwen) [inline modules/tensor_mechanics/src/actions/TensorMechanicsAction.C]: You haven't described why we need a base name here. It is not like we will have multiple stress divergence kernel sets here, or will we?

Comment (sapitts) [inline modules/tensor_mechanics/src/actions/TensorMechanicsAction.C]: Good point.  I was thinking of a multi-grain crystal plasticity model which would have separate materials, but the stress divergence will remain the same over the domain.  I'll update this code to remove the base_name.

File: modules/tensor_mechanics/src/actions/TensorMechanicsAction.C [hunk 3]
```diff
@@ -91,19 +93,15 @@ TensorMechanicsAction::act()
...
-    name << short_name;
-    name << i;
+    std::string kernel_name = "TensorMechanics" + _base_name + Moose::stringify(i);
 
     params.set<unsigned int>("component") = i;
     params.set<NonlinearVariableName>("variable") = displacements[i];
     params.set<std::vector<AuxVariableName> >("save_in") = save_in[i];
 
-    addkernel(name.str(), params);
+    addkernel(kernel_name, params);
```
Comment (sapitts) [inline modules/tensor_mechanics/src/actions/TensorMechanicsAction.C]: Is there a reason for having the addkernel as a separate method here?  The other actions all directly call
`_problem-> addKernel("StressDivergenceTensors", kernel_name, params)`

**Comment by sapitts** [comment]
My pleasure @WilkAndy. @dschwen pointed out my mistakes in using stringstream on a previous PR this week, so I thought I'd quickly implement stringify in all of our tensor mechanics actions.  The changes pass on all of the richards tests in moose, yet would you mind letting me know if the use of stringify throws an error for any of your local code.

**Comment by permcody** [comment]
:+1:

**Comment by sapitts** [comment]
Thanks @permcody !

**Comment by dschwen** [comment]
That suck-up just wants more :cake:

**Comment by WilkAndy** [comment]
It runs perfectly well on my computers, thanks @sapitts ,

a

**Pull request merged**